Applications/DisplayProperties/DisplayProperties.cpp

```diff
@@ -39,12 +39,37 @@ void DisplayPropertiesWidget::create_resolution_list()
     m_resolutions.append({ 800, 600 });
     m_resolutions.append({ 1024, 768 });
     m_resolutions.append({ 1280, 1024 });
-    m_resolutions.append({ 1366, 768 });
     m_resolutions.append({ 1440, 900 });
     m_resolutions.append({ 1600, 900 });
     m_resolutions.append({ 1920, 1080 });
     m_resolutions.append({ 2560, 1080 });
 
+    Size find_size;
+
+    bool okay = false;
+    // Let's attempt to find the current resolution and select it!
+    find_size.set_width(m_wm_config->read_entry("Screen", "Width", "1024").to_int(okay));
+    if(!okay) {
+        perror("DisplayProperties: failed to convert width to int!");
+        return;
+    }
+
+    find_size.set_height(m_wm_config->read_entry("Screen", "Height", "768").to_int(okay));
+    if(!okay) {
+        perror("DisplayProperties: failed to convert height to int!");
```

Comment:
`perror()` is a C library function that appends an explanation of the current `errno` value.
None of the preceding code sets `errno` meaningfully, so it's not appropriate to use `perror` here,

Reply:
1am mixup sorry. I'll change it to `fprintf(stderr, ...)`